src/components/SwipeableView/index.native.tsx

```diff
@@ -1,41 +1,35 @@
 import React, {useRef} from 'react';
 import {PanResponder, View} from 'react-native';
-import PropTypes from 'prop-types';
 import CONST from '../../CONST';
+import SwipeableViewComponent from './types';
 
-const propTypes = {
-    children: PropTypes.element.isRequired,
-
-    /** Callback to fire when the user swipes down on the child content */
-    onSwipeDown: PropTypes.func.isRequired,
-};
-
-function SwipeableView(props) {
+// eslint-disable-next-line react/function-component-definition
+const SwipeableView: SwipeableViewComponent = ({children, onSwipeDown}) => {
```

Comment:
I had to disable `react/prop-types` because of [this](https://stackoverflow.com/a/59402285):

> The : React.FC<ButtonProps> part was ignored by eslint

Reply:
We don't need `react/prop-types` as typescript does the same.